.github/workflows/ci.yml

```diff
@@ -199,24 +199,18 @@ jobs:
             "test_distributed_migrations",
           ]
     steps:
-      - uses: actions/checkout@v2
-        name: Checkout code
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
+      - name: Registry login
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
 
-      - name: Build snuba docker image for CI
-        uses: docker/build-push-action@v4
-        with:
-          context: .
-          push: false
-          tags: snuba-test
-          load: true
-          target: testing
-          cache-from: |
-            type=registry,ref=ghcr.io/getsentry/snuba-ci:${{ github.sha }}
-            type=registry,ref=ghcr.io/getsentry/snuba-ci:${{ needs.snuba-image.outputs.branch }}
-            type=registry,ref=ghcr.io/getsentry/snuba-ci:latest
+      - name: Pull the image
+        env:
+          IMAGE_URL: us-central1-docker.pkg.dev/sentryio/snuba/image:${{ github.event.pull_request.head.sha || github.sha }}
```

Comment:
I think you may need to use `ghcr.io/getsentry/snuba-ci:${{ github.sha }}` instead, that seems to be where the image is being pushed in step `snuba-image`

Reply:
Thanks, fixed!